global_pose_viewer:
  ros__parameters:
    anchor_lat: [
        33.98745126,
        33.98741389,
        33.98749610,
        33.98753347,
        33.98744379,
        33.98738400,
        33.98754094,
        33.98763062,
    ]
    anchor_lon: [
        132.20151193,
        132.20162009,
        132.20169220,
        132.20158404,
        132.20151193,
        132.20171924,
        132.20178233,
        132.20159305,
    ]
